glossary.asciidoc

```diff
@@ -120,22 +132,42 @@ TODO change for clarity
 HD wallet seed::
     HD wallet seed or root seed is a potentially-short value used as a seed to generate the master private key and master chain code for an HD wallet.
 
+Homestead::
+  The second development stage of Ethereum, launched in March 2016 at block #1,150,000.
+
+Ice Age::
+  A hard fork of Ethereum at block #200,000 to introduce an exponential difficulty increase, motivating a transition to Proof-of-Stake.
+
 // In case of Ethereum, perhaps, includes a blockchain explorer too?
 IDE (Integrated Development Environment)::
 	An integrated user interface that combines a code editor, compiler, runtime, and a debugger.
 
 Immutable Deployed Code Problem::
   Once a contract's (or library's) code is deployed it becomes immutable. Being able to fix possible bugs and add new features is key for the software development cycle. This represents a challenge for smart contract development.
 
+Inter exchange Client Address Protocol (ICAP)::
+  An Ethereum Address encoding that is partly compatible with the International Bank Account Number (IBAN) encoding, offering a versatile, checksummed and interoperable encoding for Ethereum Addresses. ICAP addresses can encode Ethereum Addresses or common names registered with an Ethereum name registry.
```

Comment:
Here I'd add ... ICAP addresses in Ethereum always begin with **XE**. The aim is to introduce a new IBAN country code: XE, Ethereum E prefixed with the "extended" X, as used in non-jurisdictional currencies (e.g. XBT, XRP, XCP).